tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -86,7 +94,7 @@ def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:
     return CatalogSchema(ws, table_mapping, principal_acl, backend)
 
 
-@pytest.mark.parametrize("location", ["s3://foo/bar", "s3://foo/bar/test"])
+@pytest.mark.parametrize("location", ["s3://foo/bar", "s3://foo/bar/test", "s3://foo/bar/test/1"])
 def test_create_all_catalogs_schemas_creates_catalogs(location: str):
     """Catalog 2 and 3 should be created; catalog 1 already exists."""
```

Comment:
```suggestion
    """Catalog 2-4 should be created; catalog 1 already exists."""
```

Reply:
@JCZuurmond Thanks for reviewing. Updated this.